profile/src/main/java/com/scottlogic/deg/profile/reader/JsonProfileReader.java

```diff
@@ -86,14 +85,7 @@ public Profile read(String profileJson) throws IOException {
 
 
         List<Field> fields = profileDto.fields.stream()
-            .map(fDto ->
-                new Field(
-                    fDto.name,
-                    getFieldType(fieldTypes.getOrDefault(fDto.name, fDto.type)),
-                    fDto.unique,
-                    fDto.formatting,
-                    false)
-            )
+            .map(fDto -> new Field(fDto.name, ConstraintReaderHelpers.getFieldType(fDto.type), fDto.unique, fDto.formatting,false))
```

Comment:
I think this more readable in the previous format, when split across multiple rows.

Reply:
Yeh some people do, its just a preference thing. I need this to go through asap so formatting issues should really be discussed at a later date.